build.roc

```diff
@@ -0,0 +1,144 @@
+app "build-platform-prebuilt-binaries"
+    packages {
+        cli: "https://github.com/roc-lang/basic-cli/releases/download/0.9.1/y_Ww7a2_ZGjp0ZTt9Y_pNdSqqMRdMLzHMKfdN8LWidk.tar.br",
+    }
+    imports [
+        cli.Stdout,
+        cli.Cmd,
+        cli.Task.{ Task },
+    ]
+    provides [main] to cli
+
+SupportedTarget : [
+    MacosArm64,
+    MacosX64,
+    LinuxArm64,
+    LinuxX64,
+    WindowsArm64,
+    WindowsX64,
+]
+
+mode : [DEBUG, RELEASE]
+mode = DEBUG
+
+main : Task {} I32
+main = 
+    Task.onErr run \err -> 
+        Stdout.line! (Inspect.toStr err)
+        Task.err -1
```

Comment:
Why is this -1?

Reply:
No reason, I think it should be a `1` after a quick google

Reply:
Actually, using the latest basic-cli will eliminate this code entirely which is even better 😃